docs/_components/modal.md

```diff
@@ -94,7 +97,7 @@ Toggle a modal with a link (or a link button) by using the `data-toggle="modal"`
     tabindex="-1"
     role="dialog"
     aria-labelledby="myModalLabel"
-    aria-hidden="true">
+    aria-describedby="dialog-description">
```

Comment:
Same as @Stanton 's comment, we need to make sure that a hidden modal does not get read by SRs and cannot gain KB focus. If it has `display: none` when hidden it should be fine but worth testing.

Reply:
I will use in this case aria-modal="true" (https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/dialog-modal/dialog.html)